analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/SymbolicExecutionRunnerTest.cs

```diff
@@ -337,6 +337,19 @@ End Sub
 Private Sub WithFunc(Arg As Func(Of Integer))
 End Sub");
 
+    [TestMethod]
+    public void Analyze_LocalFunction_TopLevelStatements() =>
+        VerifyTopLevelStatements(@"
+void LocalMethod(string s)
+{
+    s = null; // Noncompliant {{Message for SMain}}
+}
+");
+
+    [TestMethod]
+    public void Analyze_TopLevelStatements() =>
+        VerifyTopLevelStatements("""string s = null; // Noncompliant {{Message for SMain}}""");
```

Comment:
Should not be needed in this case anymore
```suggestion
        VerifyTopLevelStatements("string s = null; // Noncompliant {{Message for SMain}}");
```